c/tskit/trees.h

```diff
@@ -736,6 +739,30 @@ then this flag will be true.
 */
 bool tsk_treeseq_get_discrete_genome(const tsk_treeseq_t *self);
 
+/**
```

Comment:
Just realised we need to add this to the C CHANGELOG because we have added to the API.